### ISSUE TITLE
Show indicator for unsaved changes

### ISSUE THREAD

**Issue by fhemberger** [MacDownApp/macdown#215]
At the moment, you have no visible clue if the file you're editing has changed since the last save.

Other Editors show indicators, when the file has changed:

**Sublime Text** (I personally prefer that one)
![Sublime Text](https://cloud.githubusercontent.com/assets/153481/5138723/94d44c68-7153-11e4-938c-475e96ab870e.png)

**Apple's TextEdit**
![Apple's TextEdit](https://cloud.githubusercontent.com/assets/153481/5138728/a647ecf2-7153-11e4-87f6-2c6a2ccffe88.png)

**iA Writer**
![iA Writer](https://cloud.githubusercontent.com/assets/153481/5138730/b21bd26e-7153-11e4-963e-e839233c292a.png)

**Comment by franklinyu** [comment]
> Hmm, the dot in the close button is not shown for me on Mavericks … strange.

Same for El Capitan.

**Comment by uranusjr** [comment]
Ah, I know what’s going on here. The dot indicator only shows if you check “Ask to keep changes when closing documents”.

Sublime Text seems to ignore that setting, so obviously there is a way to do it, but I don’t think it’s a good idea to ignore system preferences. We should probably go for TextEdit and iAWriter’s implementation.

**Comment by Jmuccigr** [comment]
Just for the record, this works as expected for me with the "Ask…" option selected in ElCap.

**Comment by uranusjr** [comment]
Yeah, I always have that option turned on, and never realised this has anything to do with the dot indicator.

Also I just found the “— Edit” indicator in the title bar works even if the option is unchecked. Can anyone confirm? I’m on El Capitan (10.11.6).
